server/lib/localized-render.js

```diff
@@ -16,13 +16,14 @@
 'use strict';
 const path = require('path');
 const config = require('./configuration');
+const localeSuffix = config.get('public_url').endsWith('.com.cn') ? '_x_mococn' : '';
 const useLocalizedTemplates = config.get('are_dist_resources');
 
 module.exports = function (config) {
   const i18n = config.i18n;
 
   function getLocalizedTemplatePath(req, templateName) {
-    return path.join(i18n.normalizeLocale(req.locale), templateName);
```

Comment:
Yes, we set `i18n.supportedLanguages` to `["en", "zh-CN"]` in our server.

Maybe I could override it in `lib/configuration.js` when `public_url` ends with `.com.cn`, to make this more clear?

Reply:
You know what might work well is to add a new configuration item, i18n.localeSubdirSuffix in [configuration.js](https://github.com/MozillaOnline/fxa-content-server/blob/9ea9ab49084883cfa402b6a7ddaea5b51e46c7e2/server/lib/configuration.js#L250), with a default value of '', but set to `_x_mococn` for your stack. Or, alternatively, page_template_locale_subdirectory_suffix [around here](https://github.com/MozillaOnline/fxa-content-server/blob/9ea9ab49084883cfa402b6a7ddaea5b51e46c7e2/server/lib/configuration.js#L331). That way setting the suffix logic can be part of configuration instead of being hard coded within the code here.

Reply:
> ... add a new configuration item, `i18n.localeSubdirSuffix` in `configuration.js`, with a default value of `''`, but set to `'_x_mococn'` for your stack.

PR updated with this change.